geonode/base/autocomplete_light_registry.py

```diff
@@ -28,7 +28,7 @@
 from django.db.models import Q
 from geonode.security.utils import get_visible_resources
 
-from models import ResourceBase, Region, HierarchicalKeyword, ThesaurusKeywordLabel
+from .models import ResourceBase, Region, HierarchicalKeyword, ThesaurusKeywordLabel
```

Comment:
Does this change need to have occurred? 
`from models` is the same as `from .models` no?